services/galley/src/Galley/API/Public.hs

```diff
@@ -1008,6 +1010,8 @@ docs (_ ::: url) = do
   let apidoc = encode $ mkSwaggerApi (decodeLatin1 url) models sitemap
   pure $ responseLBS status200 [jsonContent] apidoc
 
+-- FUTUREWORK: /teams/api-docs does not get queried by zwagger-ui
```

Comment:
Is that the one generated by servant-swagger?  In that case I think it's added to the swagger ui differently.  (This line is stil useful, just trying to add insight.)

Reply:
Yes, that's the one and I can't find it on swagger-ui.

I think it just needs to be added [here](https://github.com/wireapp/wire-server/blob/develop/services/nginz/zwagger-ui/api-docs/resources.json), but let's do that as part of a general Swagger cleanup.

Reply:
servant-swagger generates swagger-2.0, and the rest of our swagger is 1.2.  therefore, our swagger-ui consists of two pages, and what goes where is decided based on how it is generated.  the idea was to move everything to the second page and drop the first quickly, but, you know.